docs/modules/release-notes/pages/5-5-0.adoc

```diff
@@ -12,18 +17,17 @@
 For more details on new features, see xref:ROOT:whats-new.adoc[What's new in 5.5].
 
 == Breaking Changes
-There are no breaking changes in this release.
+* For {open-source-product-name} users only - CP Subsystem is now an {Enterprise Edition}-only feature
```

Comment:
```suggestion
* For {open-source-product-name} users only - CP Subsystem is now an {Enterprise Edition}-only feature
* TPC requires `ALL_MEMBERS` cluster routing (previously smart routing) to function on clients
```

Not sure if this is the right place - but my goal is to get the point across that new `MULTI_MEMBER` routing does not work with TPC (and in 5.5.0 there is no check for this, it will be in a later patch)